### ISSUE TITLE
'High Contrast White' theme Table Header contrast 

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1037]
**Describe the bug**
The Table Header text color does not meet contrast ratio requirement of 7:1

> What version of the Fundamental Library Styles are you using?
v0.10.0-rc.1

**To Reproduce**
1. Go to https://deploy-preview-1025--fundamental-styles.netlify.app/components/table.html
2. Check the Table Header component's contrast ratio

**Expected behavior**
The table header should have a black text-color, possibly with a bolder font-weight.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81862732-f9c1b300-951e-11ea-9331-71468d48eb31.png)
Fails WCAG AAA: https://webaim.org/resources/contrastchecker/?fcolor=6A6D70&bcolor=FAFAFA

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by jacobdevera** [comment]
fixed by #1042
